roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -124,53 +124,53 @@ def required_images(task_vars):
 
         return required
 
-    def local_images(self, images, task_vars):
+    def local_images(self, images):
         """Filter a list of images and return those available locally."""
         return [
             image for image in images
-            if self.is_image_local(image, task_vars)
+            if self.is_image_local(image)
         ]
 
-    def is_image_local(self, image, task_vars):
+    def is_image_local(self, image):
         """Check if image is already in local docker index."""
-        result = self.execute_module("docker_image_facts", {"name": image}, task_vars=task_vars)
+        result = self.execute_module("docker_image_facts", {"name": image})
         if result.get("failed", False):
             return False
 
         return bool(result.get("images", []))
 
-    @staticmethod
-    def known_docker_registries(task_vars):
+    def known_docker_registries(self):
         """Build a list of docker registries available according to inventory vars."""
-        docker_facts = get_var(task_vars, "openshift", "docker")
+        docker_facts = self.get_var("openshift", "docker")
         regs = set(docker_facts["additional_registries"])
 
-        deployment_type = get_var(task_vars, "openshift_deployment_type")
+        deployment_type = self.get_var("openshift_deployment_type")
         if deployment_type == "origin":
             regs.update(["docker.io"])
         elif "enterprise" in deployment_type:
             regs.update(["registry.access.redhat.com"])
 
         return list(regs)
 
-    def available_images(self, images, registries, task_vars):
-        """Inspect existing images using Skopeo and return all images successfully inspected."""
+    def available_images(self, images, default_registries):
+        """Search remotely for images. Returns: list of images found."""
         return [
             image for image in images
-            if self.is_available_skopeo_image(image, registries, task_vars)
+            if self.is_available_skopeo_image(image, default_registries)
         ]
 
-    def is_available_skopeo_image(self, image, registries, task_vars):
+    def is_available_skopeo_image(self, image, default_registries):
         """Use Skopeo to determine if required image exists in known registry(s)."""
+        registries = default_registries
```

Comment:
Hmm, is the rename of the argument really useful if we do this on the first line?